srcpkgs/zoom/template

```diff
@@ -5,7 +5,7 @@ revision=1
 archs="x86_64"
 wrksrc=zoom
 create_wrksrc=yes
-depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative')"
+depends="$(vopt_if systemqt 'qt5 qt5-graphicaleffects qt5-imageformats qt5-quickcontrols qt5-quickcontrols2 qt5-svg qt5-script qt5-declarative') xcb-util-image xcb-util-keysyms"
```

Comment:
split into a new line, it's ok to have it over 80 chars for the vopt_if bit i guess but please split the new dependencies onto a new line.